### PR TITLE
Qlik Core customer notice version

### PR DESCRIPTION
**Pull request by sublibra**
Test page for mini version of the site.

If we decide on this way forward. I will include the source files and update the build pipeline to use the lite version instead.

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit 7531a7f34a7200b8b9ea15a31d370319591c8b8d

https://deploy-preview-567--core-website.netlify.app

**Comment by sublibra** [comment]
> Maybe use another folder for the "lite" version to be able to keep "site" for the local builds and keep it in gitignore?
> 
> Links to core.qlik.com should be changed?
> Mira should got to https://github.com/qlik-oss/core-website/blob/master/docs/services/mira.md
> License should go to https://github.com/qlik-oss/core-website/blob/master/docs/services/licenses.md
> 
> Page looks good!

Yes. If the page design is good enough I will change the build pipeline. Probably add a step where we publish from another directory. The sourcefiles is a build version of parcel which also needs to change.

**Comment by hrigner** [comment]
Nice and slim ;) 
You have to work a bit with the responsiveness in the header. The footer disappears, but that's maybe how you want it to be? (though according to accessibility all content should be reachable regarding of device).

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,24 +1,19 @@
...
       - run:
           name: Build the site
-          command: ./build.sh
+          command: ./build-deprecation-notice.sh
       - store_artifacts:
           path: ./site
       - run:
           name: Check if external links are broken
-          command: .circleci/check-external-links.sh
+          command: .circleci/check-external-links.sh
```
Comment (hrigner) [inline .circleci/config.yml:16]: many spaces..

**Pull request merged**